scripts/presubmit.sh

```diff
@@ -74,6 +74,14 @@ main() {
     grep -v _string.go | \
     tr '\n' ' ')"
 
+  # Prevent the creation of proto files with .txt extensions.
+  bad_protos="$(find . -name '*.pb.txt' -o -name '*.proto.txt')"
```

Comment:
I imagine people could very well come up with other variations, but this is a good starting point.

Reply:
Yep, these are the two that could cause import problems. I'll do the other repo script similarly next.